.github/ISSUE_TEMPLATE.md

```diff
@@ -1,13 +1,21 @@
+Please, check https://docs.docker.com/docker-for-win/troubleshoot/.
+Issues without logs and details cannot be exploited, and will be closed.
+
+  - [ ] I have tried with the latest version of my channel (Stable or Edge)
+  - [ ] I have submitted Diagnostics
+  - [ ] I have included the Diagnostics ID in this Issue
+
```

Comment:
we should probably ad a field here for the windows and docker for windows version as well

Reply:
Good idea!